target/linux/x86/image/Makefile

```diff
@@ -49,6 +49,9 @@ BOOTOPTS:=$(call qstrip,$(CONFIG_GRUB_BOOTOPTS))
 define Build/combined
 	$(CP) $(KDIR)/$(KERNEL_NAME) $@.boot/boot/vmlinuz
 	-$(CP) $(STAGING_DIR_ROOT)/boot/. $@.boot/boot/
+	$(INSTALL_DIR) $@.boot/boot/grub
```

Comment:
1. This changed doesn't belong to this PR, it has nothing to do with EFI etc.
2. This doesn't belong to the combined image, it's related to grub-config or grub-install